src/core/primitives.js

```diff
@@ -203,14 +203,14 @@ class Dict {
     return this._map.has(key);
   }
 
-  forEach(callback) {
-    for (const [key, value] of this._map) {
-      callback(
+  *[Symbol.iterator]() {
+    for (const [key, val] of this._map) {
```

Comment:
What's the reason for renaming `value` to `val` here, given that we seem to use `value` elsewhere in e.g. the files touched in this commit?

Reply:
I don't know, these patches are a few years old at this point in time :-)